manifests/plugin/curl_json.pp

```diff
@@ -5,6 +5,7 @@
   $keys,
   $user     = undef,
   $password = undef,
+  $typesdb = undef,
```

Comment:
I think this equal sign not being lined up with the rest will trigger a puppet-lint error